### ISSUE TITLE
Dependabot `pyjwt` conflict: Key confusion through non-blocklisted public key formats

### ISSUE THREAD

**Issue by BlakeJC94** [allegroai/clearml#692]
Hello,

I need to address this vulnerability raised by Dependabot on a repo that uses ClearML:
```
Key confusion through non-blocklisted public key formats #23

Upgrade pyjwt to fix 1 Dependabot alert in requirements-dev.txt

Upgrade pyjwt to version 2.4.0 or later. For example:

pyjwt>=2.4.0
```

However this is incompatible with `ClearML==1.4.1` since the latest release requires `pyjwt<=2.1.0`.

I can see `requirements.txt` has been updated a couple of weeks ago. Are there any upcoming plans to release a patch to ClearML that addresses this vulnerability in the near future?

**Comment by DavidNativ** [comment]
Hello,
A fix was already committed and is available in the latest RC (1.4.2rc1, I think). We will release 1.5 soon.

**Comment by BlakeJC94** [comment]
Excellent, thanks for the update!
